computestorage/format.go

```diff
@@ -0,0 +1,24 @@
+package computestorage
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Microsoft/hcsshim/internal/oc"
+	"go.opencensus.io/trace"
+	"golang.org/x/sys/windows"
+)
+
+// FormatWritableLayerVhd formats a virtual disk for use as a writable container layer.
```

Comment:
I think it would be good to also include a comment here saying that the vhd will be mounted if it is not already mounted. Same thing for other functions which take a vhd handle as input.

Reply:
Good call, adding